src/generator/app.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var express = require('express');
+var app = express();
+
+var generator = require('./backend/generator.js');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Combine this with the previous 'var' statement.](https://www.codacy.com/app/triplez/open-event-webapp/file/2708752417/issues/source?bid=3233857&fileBranchId=3342834#l5)